docs/index.md

```diff
@@ -3,174 +3,26 @@
 page_title: "ocm Provider"
 subcategory: ""
 description: |-
+  
 ---
-# OCM Provider
 
-## Introduction
```

Comment:
I'm uncertain if we want to remove all these. When you generate docs with `tfplugindocs` it automatically removes these, since they are not really a part of the provider code.

Perhaps this file needs to be handled manually.